site/content/learn/headless/generating-pdfs.md

```diff
@@ -56,12 +56,12 @@ We can also have custom headers and footers added to our pages, displaying value
 
 {{< tabs "2" >}}
 {{< tab "Playwright" >}}
-```js {10-11,21-31}
+```js {hl_lines=["11-12","21-31"]}
 {{< readfile filename="samples/playwright/pdf-hd.js" >}}
 ```
 {{< /tab >}}
 {{< tab "Puppeteer" >}}
-```js {10-11,13,23-33}
```

Comment:
@pilimartinez I think this highlight should be moved up a couple of lines to include the whole block